common/genes/05_genes_special_accessories_clothes.txt

```diff
@@ -1551,6 +1551,76 @@ special_genes = {
 				boy = male
 				girl = female
 			}
+			
+			### Dwarven ###				# Wildhammer and Frostborn wear all # Old Dwarven, Bronzebeard and Dark Iron wear armour
+			dwarven_commoner_clothes = {
```

Comment:
I don't see what is the point of creating a new set of clothes.
So we want the Wildhammer dwarves to wear the norse everything. In this case, we just add the norse graphical cultures to the Wildhammer.

Reply:
I can assign wildhammer/frostborn the norse graphical culture for all social classes and add `portrait_dwarven_clothing_trigger` to string `northern_war` (line 2556 in 01_clothes.txt ). I think that should bypass needing to create a new set of cultural clothing right? The reason behind making a new set was greater customisability (I saw it existed for some other warcraft races) but I can try adding triggers using the method above to add/change accessories; it feels cleaner to have a set dedicated to each culture instead of inserting into the base game sets. Is the mod size the concern here?

Reply:
> I can assign wildhammer/frostborn the norse graphical culture for all social classes and add portrait_dwarven_clothing_trigger to string northern_war (line 2556 in 01_clothes.txt ). I think that should bypass needing to create a new set of cultural clothing right?

Yes.

> The reason behind making a new set was greater customisability (I saw it existed for some other warcraft races) but I can try adding triggers using the method above to add/change accessories; it feels cleaner to have a set dedicated to each culture instead of inserting into the base game sets. Is the mod size the concern here?

No, it's because it's a vanilla set copy.